cityofaustin/atd-moped
# Testing the pro version

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/16860

I initially opened this PR to test what the pro version was like before we bought it. But now we have it. 


## Testing

https://deploy-preview-1303--atd-moped-main.netlify.app/moped/projects


**Steps to test:**

you can pin columns to the left or the right, you can resize columns.

You can view 1000 rows per page.

Try to break the table 


If you want to test locally, check out and `npm install`.
create an .env file in the moped-editor folder with the entry `REACT_APP_MUIX_LICENSE_KEY=`, the key is in 1password under MUI. 
 

---
#### Ship list
